app/gui/src/dashboard/pages/dashboard/Dashboard.tsx

```diff
@@ -164,28 +160,6 @@ function DashboardInner(props: DashboardProps) {
     }
   }, [openEditor, openProject, categoriesAPI])
 
-  React.useEffect(
```

Comment:
@somebody1234 I removed this effect because it appears we don't need that one? 

And it's the only place where we use `updateModal`

Reply:
maybe. but if dialogs natively handle `Escape`, we should fully remove the `closeModal` binding
(i think this is the only place where the `closeModal` binding is handled)

Reply:
Yep, RAC has it's own esc press handling

Reply:
This 'might' affect the legacy modals, idk how they handle esc press